blocks/log/json.py

```diff
@@ -0,0 +1,54 @@
+from mimir import Logger
+
+from .log import TrainingLogBase
+
+
+class PiclableLogger(object):
+    def __init__(self, filename, **kwargs):
+        logger = Logger(**kwargs)
+        self.__dict__ = logger.__dict__
+        self.all_kwargs = kwargs
+        self.all_kwargs['filename'] = filename
+        self.status = {}
+        self.iteration_status = {}
+
+    def __setstate__(self, state):
+        logger = Logger(**state)
+        self.__dict__ = logger.__dict__
+
+    def __getstate__(self):
+        self.flush()
```

Comment:
I assume, that we pickle the logger and finish the iteration. Otherwise, it will break (it will create a new line in the jsonl file).